code/lib/cli/src/repro-templates.ts

```diff
@@ -86,7 +86,7 @@ export const allTemplates: Record<string, Template> = {
   },
   'react-vite/default-js': {
     name: 'React Vite (JS)',
-    script: 'yarn create vite . --template react',
+    script: 'yarn create vite@beta . --template react',
```

Comment:
@IanVS This is awesome!!!  So happy about this 🎉 

Adding `@beta` is great for testing, but should stay as `yarn create vite` when we merge

We should have a separate `yarn create vite@beta` that only runs at the `daily` cadence. We can probably do this for TS only (since it's generally a superset of JS) and do it for all frameworks.

WDYT? @yannbf @kasperpeulen

Reply:
Agreed!

Reply:
I'm not so sure keeping @beta in the tests will be helpful.  I don't think we need to officially support beta after the final release (likely tomorrow), and we can always add it back in before the next major change.  I'd recommend at least deferring until then, personally.

Reply:
So your recommended approach is that we test against the beta now, for the next couple of days. And when Vite 4.0 has been released, we go back to testing against stable.
And we do the same thing when Vite 5.0 is getting close, etc.

That sounds fair to me.

I think there's CI iteration 2 in the future where we have a setup that tests against all the betas continuously, (so eg. we would have caught Angular 15 breaking before it was released), and that system would likely supersede the work here.

Reply:
We could leave it in for a few days, or I've already taken it out of this PR.  I think it's a low-risk item right now, especially since we showed here in this PR that it doesn't cause any trouble in any of the frameworks.

I'm also working on getting storybook back into vite-ecosystem-ci, which would mean that storybook would be continuously tested against unreleased versions of vite, which would be ideal.